contracts/test-utils/src/mocha_blockchain.ts

```diff
@@ -97,6 +99,10 @@ export const describe = _.assign(mochaDescribe, {
         const describeCall = process.env.TEST_ALL ? mochaDescribe : mochaDescribe.skip;
         return describeCall(description, callback);
     },
+    fork(description: string, callback: SuiteCallback): ISuite | void {
```

Comment:
I wonder if this should have a signature like `fork(description, opts, callback)` so we can pass unlocked accounts and fork block numbers in the tests. I feel like the tests should own these parameters. Thoughts?

Reply:
I'd like something like that, but it's not actually very easy to instantiate a new provider for `blockchainTests` without a large-ish refactor. Let's backlog it?